src/bidiMapper/domains/context/browsingContextProcessor.ts

```diff
@@ -237,6 +237,21 @@ export class BrowsingContextProcessor {
     );
   }
 
+  async process_browsingContext_captureScreenshot(
+    params: BrowsingContext.CaptureScreenshotParameters
+  ): Promise<BrowsingContext.CaptureScreenshotResult> {
+    const context = this.#browsingContextStorage.getKnownContext(
+      params.context
+    );
+    // TODO: return context.captureScreenshot(params);
```

Comment:
Correct. And `context.captureScreenshot` in turn should call `await this.#cdpClient.sendCommand('Page.captureScreenshot', {})`